assets/components/src/steps-list/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Steps List
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies.
+ */
+import { StepsListItem } from '../';
+import './style.scss';
+
+/**
+ * External dependencies.
+ */
+import classnames from 'classnames';
+
+class StepsList extends Component {
+	/**
+	 * Render
+	 */
+	render() {
+		const { className, stepsListItems, narrowList, style = {} } = this.props;
+		const classes = classnames( 'steps-list', className, narrowList && 'steps-list__narrow-list' );
+
+		return (
+			<div className={ classes } style={ style }>
+				{ stepsListItems.map( ( listItem, index ) => (
+					<StepsListItem key={ index } listItemCount={ index + 1 } listItemText={ listItem } />
```

Comment:
They `key` attribute on a component rendered in a `map` function must be any unique value. Since `listItems` is just an array of strings, there was no `listItem.id` property, so we can just use the `index` instead.